intensional-reasoning-test.scm

```diff
@@ -56,8 +56,8 @@
 (define X (Variable "$X"))
 (define Y (Variable "$Y"))
 ;; (define target (IntensionalInheritance X Y))
-;; (define target (IntensionalSimilarity X Y))
-(define target (IntensionalDifference X Y))
+(define target (IntensionalSimilarity X Y))
```

Comment:
I note that here you've replaced `IntensionalDifference` by `IntensionalSimilarity`, which should work too though.